### PR TITLE
Allow rewinding of sequences

### PR DESCRIPTION
**Pull request by composerinteralia**
A possible solution for https://github.com/thoughtbot/factory_girl/issues/80

### CODE REVIEW

**Comment by composerinteralia** [comment]
That's a good dog bot! houndci-bot is still less picky than my dog.

**Comment by composerinteralia** [comment]
@joshuaclayton hound is recommending %i, but that syntax wasn't introduced until 2.0.0. Travis is still running against 1.9.3.

File: spec/acceptance/sequence_resetting_spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require "spec_helper"
+
+describe "FactoryGirl.reset_sequences" do
```
Comment (composerinteralia) [inline spec/acceptance/sequence_resetting_spec.rb]: oops, typo. I will fix that

**Comment by composerinteralia** [comment]
Rebased on `master` to user the new name

**Comment by damln** [comment]
Hi guys!
I'm working on a similar feature based on this branch.
Any updates from @joshuaclayton or anyone in the core team?

🥇

**Comment by joshuaclayton** [comment]
Thanks @composerinteralia! I've merged this as 50eeb67241ea78a6b138eea694a2a25413052f49

@damln This is good to go, and has been merged. I'll get some documentation into the GETTING_STARTED doc, but if you point to the master branch, you should be good to go 😄

**Comment by jsmartt** [comment]
@joshuaclayton, any chance we can get a release with this (and the other latest changes)?

**Pull request closed without merging**